cfme/tests/configure/test_multi_tenancy_manual.py

```diff
@@ -0,0 +1,438 @@
+# -*- coding: utf-8 -*-
+# pylint: skip-file
+"""Manual tests"""
+
+import pytest
+
+from cfme import test_requirements
+
+pytestmark = [pytest.mark.ignore_stream('5.9', '5.10', 'upstream')]
+
+
+@pytest.mark.manual
+@test_requirements.cfme_tenancy
+def test_tenant_unique_catalog():
```

Comment:
Lots of opportunity for automation here :)

Reply:
Yes, automating these test cases is on my TODO list.